docs/podman-search.1.md

```diff
@@ -0,0 +1,107 @@
+% podman(1) podman-search - Tool to search registries for an image
+% Urvashi Mohnani
+# podman-search "1" "January 2018" "podman"
+
+## NAME
+podman search - Search a registry for an image
+
+## SYNOPSIS
+**podman search**
+**TERM**
+[**--filter**|**-f**]
+[**--format**]
+[**--limit**]
+[**--no-trunc**]
+[**--registry**]
+[**--help**|**-h**]
+
+## DESCRIPTION
+**podman search** searches a registry or a list of registries for a matching image.
+The user can specify which registry to search by setting the **--registry** flag, default
+is the default registries set in the config file.
+The number of results can be limited using the **--limit** flag. If more than one registry
+is being searched, the limit will be applied to each registry. The output can be filtered
+using the **--filter** flag.
+
+**podman [GLOBAL OPTIONS]**
+
+**podman search [GLOBAL OPTIONS]**
+
+**podman search [OPTIONS] TERM**
+
+## OPTIONS
+
+**--filter, -f**
+Filter output based on conditions provided (default [])
+
+Supported filters are:
+- stars (int - number of stars the image has)
+- is-automatec (boolean - true | false) - is the image automated or not
+- is-official (boolean - true | false) - is the image official or not
```

Comment:
what makes an image "official".  A bit more verbiage here or the opening description would be nice.

Reply:
Of course this only means that someone paid Docker to make it official...

Reply:
Should I add a description or let it be?

Reply:
Docker does not describe it and it is for the benefit of  Docker to call things official.  So I would not describe it.  I am not sure if we declare Red Hat images as Official.  So lets leave this alone.